Core/src/main/java/com/plotsquared/core/util/StringMan.java

```diff
@@ -320,4 +325,47 @@ public static boolean contains(String name, char c) {
         return col;
     }
 
+    /**
+     * @param message an input string
+     * @return a list of strings
+     * @since TODO
+     *         <p>
+     *         Converts multiple quoted and single strings into a list of strings e.g.
+     *         <p>
+     *         <table border="1">
```

Comment:
```suggestion
     *         <table border="1">
     *         <caption>Converts multiple quoted and single strings into a list of strings</caption>
```
DocLint wants a caption, but that format looks much better to work with
![](https://i.imgur.com/Mwpk0Hm.png)